### DISCUSSION TITLE
UseWhiteList and msstore Question

### DISCUSSION THREAD

**Discussion by Frank-Radmayr** [Romanitho/Winget-AutoUpdate#659] [Q&A]
Hi all
I am new to Winget-AutoUpdate and am testing some things, I have a few questions.
**UseWhiteList**:
If I subsequently create entries locally on the client in the file "included_apps.txt", these are not recognized when the Scheduled Task "Winget-AutoUpdate" is executed and therefore the application is not updated.
What do I have to do so that the new entries are also recognized?

**msstore**:
As I have read, only the winget store is queried, is that correct?
If I have installed an app via msstore, will it not be updated?
Will it be updated if the install source is also available on the winget store?

Thank you for your help so that I can expand my knowledge here.
Kind regards Frank

**Comment by Romanitho** [comment]
Hi and welcome,

to use whitelist, you need an included_apps.txt file and enable UseWhiteList during install (or use the gui).

We do not manage msstore apps as they are automatically maintained by the store.

**Reply by Frank-Radmayr** [reply]
Hi Romanitho
Thank you for your quick reply.
As you can see on my printscreen, I already have the file included_apps.txt and UseWhiteList active.
The update for KeePass (DominikReichl.KeePass) also worked, but the WinGet-Autoupdate doesn't seem to recognize the new entries for Spotify that I added later.

![image](https://github.com/user-attachments/assets/eaeb11b1-03e6-4157-833d-d1f1e850299e)

Regarding msstore
It would be nice if these app updates could also be coordinated via Winget-Autoupdate (WAU).
We create SW package with Powershell scripts in which the store apps are installed with the winget.exe command and integrate them in SCCM and then distribute them to our clients via SCCM.
As most apps from the winget store require admin rights during installation, we only use apps from the msstore, as these are installed in the user context.
If these "WinGet SW packages" are then installed on the clients, the update will then take place automatically and we would also like to control this, e.g. with WAU.

**Reply by Romanitho** [reply]
Spotify is probably installed on your profile. Enable user context option.
Then, I would recommend installing apps in the system context. WAU handles admin context for updates.

**Reply by KnifMelti** [reply]
[Scope: user](https://github.com/microsoft/winget-pkgs/blob/master/manifests/s/Spotify/Spotify/1.2.9.746.gbc57b7ae/Spotify.Spotify.installer.yaml)
